src/DotNet.Status.Web/.config/settings.json

```diff
@@ -49,6 +49,36 @@
           "Assignee": "riarenas",
           "IssuesId": "dotnet-arcade"
         },
+        {
+          "Project": "internal",
```

Comment:
Should these have an assignee as well?

Reply:
I opted for "no". I was concerned a single assignee would break the system when that person goes on vacation. But if these don't get triaged to FR or Operations consistently enough then a single assignee might help.

Reply:
I think as long as the issue is opened in arcade we'll end up triaging these to FR pretty quickly, guess the concern is about whether build monitor can open the issues without an assignee.

Reply:
Entries below this have no assignee, so I think it'll be fine (as long as it's not an either/or thing with the labels).